@navikt/core/css/form/textarea.css

```
@@ -68,7 +59,7 @@
 }
 
 .navds-textarea__counter {
-  margin-top: var(--a-spacing-05);
+  margin-top: -0.375rem;
```

Vet ikke om dette er verdt den unødvendige logikken, så kan ignoreres
```suggestion
  margin-top: calc(var(--a-spacing-1-alt) * -1)
```

---

Vi har ikke gjort det andre steder, så hvis vi skal endre det en dag bør vi kanskje endre alle steder.